wagtail/contrib/postgres_search/utils.py

```diff
@@ -60,17 +60,29 @@ def get_descendant_models(model):
     return descendant_models
 
 
-def get_descendants_content_types_pks(model):
+def get_content_type_pk(model):
+    # We import it locally because this file is loaded before apps are ready.
+    from django.contrib.contenttypes.models import ContentType
+    return ContentType.objects.get_for_model(model).pk
+
+
+def get_ancestors_content_types_pks(model):
+    """
+    Returns content types ids for the descendants of this model, excluding it.
+    """
     from django.contrib.contenttypes.models import ContentType
     return [ct.pk for ct in
-            ContentType.objects.get_for_models(*get_descendant_models(model))
-            .values()]
+            ContentType.objects.get_for_models(*model._meta.parents).values()]
```

Comment:
IIRC ``model._meta.parents`` only gets the immediate parents of this model, not all ancestors